Dockerfile

```diff
@@ -15,9 +17,7 @@ RUN yarn remove 'typescript' --dev && yarn add 'typescript'
 RUN yarn install --production --frozen-lockfile
 RUN chmod +x distribution/commands/danger.js
 
-
-FROM node:18-slim
-WORKDIR /usr/src/danger
```

Comment:
Was deleting this `WORKDIR` line intentional?

Reply:
it is not deleting, it's setup in the parent layer. see the full `Dockerfile` to see the context.